packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockUnicodeRunsLength.st

```diff
@@ -0,0 +1,4 @@
+emojis
+mockUnicodeRunsLength
+
+	^ 3
```

Comment:
This method returns the max run length right? So would `mockMaxRunLength` be a more appropriate name?

Reply:
no, this is the length of the specific run created by `mockUnicodeRun1`